packages/app/src/cli/services/context.test.ts

```diff
@@ -382,6 +382,61 @@ dev_store_url = "domain1"
     })
   })
 
+  test('shows the correct banner content when running for the first time with config file', async () => {
+    await inTemporaryDirectory(async (tmp) => {
+      // Given
+      vi.mocked(getAppInfo).mockReturnValue(undefined)
+      vi.mocked(load).mockReset()
+      vi.mocked(load).mockResolvedValue(
+        testApp({
+          configurationPath: joinPath(tmp, 'shopify.app.dev.toml'),
+          configuration: {
+            client_id: APP2.apiKey,
+            name: APP2.title,
+            scopes: 'read_products',
+            application_url: 'https://my-apps-url.com',
+            auth: {
+              redirect_urls: ['https://my-apps-url.com/auth/shopify'],
+            },
+          },
+        }),
+      )
+      vi.mocked(fetchAppFromApiKey).mockResolvedValueOnce(APP2)
+      vi.mocked(fetchStoreByDomain).mockResolvedValue({organization: ORG1, store: STORE1})
+
+      // When
+      await ensureDevContext(
+        {
+          directory: 'app_directory',
+          reset: false,
+          config: 'dev',
+        },
+        'token',
+      )
+
+      // Then
+      expect(renderInfo).toHaveBeenCalledWith({
+        body: [
+          {
+            list: {
+              items: [
+                `Org:          ${ORG1.businessName}`,
+                `App:          ${APP2.title}`,
+                `Dev store:    ${STORE1.shopDomain}`,
+                'Update URLs:  Not yet configured',
+              ],
+            },
+          },
+          '\nTo reset your default dev config, run',
+          {
+            command: 'npm run dev -- --reset',
+          },
+        ],
+        headline: 'Using these settings:',
```

Comment:
Could we change the headline to show the config file, to be more explicit?
```suggestion
        headline: `Using config file ${configFileName}:`,
```

Reply:
I was thinking that we could add it to the table?